<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_settings</class>
 <widget class="QDialog" name="dialog_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_msaa">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>82</height>
      </size>
     </property>
     <property name="title">
      <string>Multisampling / Anti-aliasing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QSlider" name="hSlider_antialiasing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="hSlider_multisampling">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_antialiasing">
        <property name="text">
         <string>Anti-aliasing</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_multisampling">
        <property name="text">
         <string>Multisampling</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_difficulty">
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>82</height>
      </size>
     </property>
     <property name="title">
      <string>Difficulty </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox_navigation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Navigation (mid points)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox_settings">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_spaceship">
     <property name="title">
      <string>Space ship</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_smallship">
        <property name="text">
         <string>Small ship</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_alienship">
        <property name="text">
         <string>Alien ship</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_ownship">
        <property name="text">
         <string>Own ship</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_wormhole">
     <property name="title">
      <string>Wormhole</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_triangles">
        <property name="text">
         <string>Triangles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_quads">
        <property name="text">
         <string>Quads</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_multisampling</tabstop>
  <tabstop>checkBox_antialiasing</tabstop>
  <tabstop>hSlider_antialiasing</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_multisampling</sender>
   <signal>toggled(bool)</signal>
   <receiver>hSlider_multisampling</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_antialiasing</sender>
   <signal>toggled(bool)</signal>
   <receiver>hSlider_antialiasing</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
